src/Apps/Legion/Components/FlowPicker/FlowPicker.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useCallback } from 'react';
+import {
+    DefaultButton,
+    Stack,
+    classNamesFunction,
+    styled
+} from '@fluentui/react';
+import { useExtendedTheme } from '../../../../Models/Hooks/useExtendedTheme';
+import { getDebugLogger } from '../../../../Models/Services/Utils';
+import {
+    IFlowPickerProps,
+    IFlowPickerStyleProps,
+    IFlowPickerStyles
+} from './FlowPicker.types';
+import { getStyles } from './FlowPicker.styles';
+import {
+    AppNavigationContextActionType,
+    AppPageName
+} from '../../Contexts/NavigationContext/AppNavigationContext.types';
+import { useAppNavigationContext } from '../../Contexts/NavigationContext/AppNavigationContext';
+import { WizardStepNumber } from '../../Contexts/WizardNavigationContext/WizardNavigationContext.types';
+
+const debugLogging = false;
+const logDebugConsole = getDebugLogger('FlowPicker', debugLogging);
+
+const getClassNames = classNamesFunction<
+    IFlowPickerStyleProps,
+    IFlowPickerStyles
+>();
+
+const FlowPicker: React.FC<IFlowPickerProps> = (props) => {
+    const { styles } = props;
+
+    // contexts
+    const { appNavigationDispatch } = useAppNavigationContext();
+
+    // state
+
+    // hooks
+
+    // callbacks
+    const navigate = useCallback(
+        (step: WizardStepNumber) => {
+            appNavigationDispatch({
+                type: AppNavigationContextActionType.NAVIGATE_TO,
+                payload: {
+                    pageName: AppPageName.Wizard,
+                    step: step
+                }
+            });
+        },
+        [appNavigationDispatch]
+    );
+
+    // side effects
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useExtendedTheme()
+    });
+
+    logDebugConsole('debug', 'Render');
+
+    return (
+        <Stack className={classNames.root}>
+            <DefaultButton
+                text={'Discover'}
+                onClick={() => navigate(WizardStepNumber.AddSource)}
+            />
+            <DefaultButton
+                text={'Modify'}
+                onClick={() => navigate(WizardStepNumber.Modify)}
+            />
+            <DefaultButton
+                text={'View'}
+                onClick={() => navigate(WizardStepNumber.Save)}
+            />
+        </Stack>
+    );
+};
+
+export default styled<
```

Comment:
Should we keep making styled components, since we now know we'll eventually have to remove them to transition to Fluent 9?

Reply:
Generally yes. I think so. Unless we want to swap back to the mergeStyles pattern since that seems closer to the V9 patterns and less of a migration headache. I'm open to ideas and we aren't likely to need to build very many generalized components that need to take in customized styles as props which is hte main benefit you get from styled components so I could be convinced. 🤔